source/exec.tex

```diff
@@ -5672,3 +5690,845 @@
+and
+\item \tcode{set_stopped_t()}.
+\end{itemize}
+
+\pnum
+\tcode{allocator_type} shall meet the \oldconcept{Allocator}
+requirements\iref{allocator.requirements.general}.
+
+\rSec3[task.members]{\tcode{task} Members}
+
+\begin{itemdecl}
+task(task&& other) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes \exposid{handle} with \tcode{exchange(other.\exposid{handle},
+\{\})}.
+\end{itemdescr}
+
+\begin{itemdecl}
+~task();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+
+\begin{codeblock}
+if (@\exposid{handle}@)
+  @\exposid{handle}@.destroy();
+\end{codeblock}
+\end{itemdescr}
+
+\begin{itemdecl}
+template<@\libconcept{receiver}@ R>
+@\exposid{state}@<R> connect(R&& recv);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{bool(\exposid{handle})} is \tcode{true}.
+
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{state}<R>(exchange(\exposid{handle},
+\{\}), std::forward<R>(recv));}
+\end{itemdescr}
+
+\rSec3[task.state]{\tcode{Class template \tcode{task::state}}}
+
+\begin{codeblock}
+namespace std::execution {
+  template<class T, class Environment>
+    template<@\libconcept{receiver}@ Rcvr>
+  class task<T, Environment>::@\exposid{state}@ { // \expos
+  public:
+    using operation_state_concept = operation_state_t;
+
+    template<class R>
+    @\exposid{state}@(coroutine_handle<promise_type> h, R&& rr);
+    ~@\exposid{state}@();
+    void start() & noexcept;
+
+private:
+    using @\exposidnc{own-env-t}@ = @\seebelow@;     // \expos
+    coroutine_handle<promise_type> @\exposidnc{handle}@;  // \expos
+    remove_cvref_t<Rcvr>              @\exposidnc{rcvr}@;    // \expos
+    @\exposid{own-env-t}@                      @\exposidnc{own-env}@; // \expos
+    Environment                    @\exposidnc{environment}@; // \expos
+  };
+}
+\end{codeblock}
+
+\begin{itemdescr}
+\pnum
+The type \exposid{own-env-t} is \tcode{Environment::template
+env_type<decltype(get_env(\linebreak declval<Rcvr>()))>} if that % avoid Overfull
+qualified-id is valid and denotes a type, \tcode{env<>} otherwise.
+\end{itemdescr}
+
+\begin{itemdecl}
+template<class R>
+@\exposid{state}@(coroutine_handle<promise_type> h, R&& rr);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes
+
+\item \exposid{handle} with \tcode{std::move(h)};
+\item \exposid{rcvr} with \tcode{std::forward<R>(rr)};
+\item \exposid{own-env}
+    with \tcode{\exposid{own-env-t}(get_env(\exposid{rcvr}))} if that expression
+    is valid and \tcode{\exposid{own-env-t}()} otherwise.
+    If neither of these expressions is valid, the program is ill-formed.
+\item \exposid{environment} with \tcode{Environment(\exposid{own-env})} if that expression is
+    valid, otherwise \tcode{Environment(get_env(\exposid{rcvr}))}
+    if this expression is valid, otherwise \tcode{Environment()}.
+    If neither of these expressions is valid, the program is ill-formed.
```

Comment:
There is no `itemize` environment surrounding these.

I'm surprised it even builds, but I guess that's a typical TeX moment.